xwiki-platform-core/xwiki-platform-search/xwiki-platform-search-solr/xwiki-platform-search-solr-api/src/main/java/org/xwiki/search/solr/internal/metadata/AttachmentSolrMetadataExtractor.java

```diff
@@ -92,6 +92,9 @@ public boolean setFieldsInternal(LengthSolrInputDocument solrDocument, EntityRef
 
         setLocaleAndContentFields(attachment, solrDocument);
 
+        // Extract more metadata
```

Comment:
Out of curiosity, are the additional metadata?

Reply:
That's where we inject the links in the Solr document currently, but the idea was also to provide an extension point for extensions to impact the Solr search code indexing (an example of extension that would benefit from it is https://extensions.xwiki.org/xwiki/bin/view/Extension/Solr%20Security/).

Reply:
So it's not anymore where we inject the links, I made that part a bit more core to the Solr indexer. But it's still very useful for extensions, so I will keep it even if it became useless strictly for XWIKI-16192 (OK I'm too lazy to revert it and redo that in a different branch).